sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -328,6 +330,17 @@ public class Configs {
     private static final String HTTP_CONNECTION_WITHOUT_TLS_ALLOWED = "COSMOS.HTTP_CONNECTION_WITHOUT_TLS_ALLOWED";
     private static final String HTTP_CONNECTION_WITHOUT_TLS_ALLOWED_VARIABLE = "COSMOS_HTTP_CONNECTION_WITHOUT_TLS_ALLOWED";
 
+    // Config to indicate whether hostname validation for TLS connections to the Cosmos DB endpoints
+    // (Gateway and Backend) should be disabled
```

Comment:
```suggestion
    // (Backend) should be disabled
```

Reply:
yea but if the customer say disable it, it can not really be disabled for gateway, that is why the ask